packages/sidecar/src/cli/help.ts

```diff
@@ -1,4 +1,5 @@
 import { EVERYTHING_MAGIC_WORDS, NAME_TO_TYPE_MAPPING } from "../cli/tail.js";
+import { VALID_FORMATTERS } from "../formatters/types.js";
```

Comment:
```suggestion
import { AVAILABLE_FORMATTERS } from "../formatters/types.js";
```

Available would be better than "valid"